### PR TITLE
Correct source detection for lambda logs

### PR DESCRIPTION
**Pull request by NBParis**
Lambda logs were flagged as cloudwatch until now

### CODE REVIEW

File: Log/lambda_function.py [hunk 1]
```diff
@@ -142,8 +142,8 @@ def awslogs_handler(s, event):
     # Get logs
     data = zlib.decompress(base64.b64decode(event["awslogs"]["data"]), 16 + zlib.MAX_WBITS)
     logs = json.loads(str(data))
-    metadata[DD_SOURCE] = "cloudwatch"
-
+    source = logs["logGroup"]
```
Comment (tmichelet) [inline Log/lambda_function.py]: is `logGroup` always present? Would it be safer to use `logs.get("logGroup", "DEFAULTVALUE")`?

Comment (NBParis) [reply Log/lambda_function.py]: It should always be there from cloudwatch logs but will change as you suggest for safety.

**Comment by NBParis** [comment]
@tmichelet updated (added the kinesis log source at the same time)

**Comment by tmichelet** [comment]
👍

**Pull request merged**